voter-dapp/src/ResolvedRequests.js

```diff
@@ -0,0 +1,85 @@
+import React, { useMemo } from "react";
+import { drizzleReactHooks } from "drizzle-react";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+import Typography from "@material-ui/core/Typography";
+
+import { formatDate } from "./common/FormattingUtils.js";
+
+function ResolvedRequests() {
+  const { drizzle, useCacheCall, useCacheEvents } = drizzleReactHooks.useDrizzle();
+  const { web3 } = drizzle;
+
+  const currentRoundId = useCacheCall("Voting", "getCurrentRoundId");
+
+  const { account } = drizzleReactHooks.useDrizzleState(drizzleState => ({
+    account: drizzleState.accounts[0]
+  }));
+
+  const resolvedEvents =
+    useCacheEvents(
+      "Voting",
+      "PriceResolved",
+      useMemo(() => ({ filter: { resolutionRoundId: currentRoundId - 1 }, fromBlock: 0 }), [currentRoundId])
```

Comment:
Here I queried requests from the previous round, but we can do this differently. 

If we want to support pagination, we should be able to query a range by using `fromBlock` and `toBlock`. If we want to just query the last few rounds, we can use the `filter` syntax. Both options should perform reasonably since `roundId` is indexed. A third option is to fetch all past events, but I think we want to avoid unbounded results.

Reply:
I think this is a good tradeoff for now. Long term, I think it might make sense to allow people to selectively query a round or range of rounds if they really want more data.